apps/web-giddh/src/app/invoice/invoice.component.ts

```diff
@@ -33,34 +33,39 @@ export class InvoiceComponent implements OnInit, OnDestroy {
 
     public ngOnInit() {
         combineLatest([this._activatedRoute.params, this._activatedRoute.queryParams])
-            .pipe(auditTime(700))
+            .pipe(takeUntil(this.destroyed$))
             .subscribe(result => {
                 let params = result[0];
                 let queryParams = result[1];
 
                 if (params) {
-                    if (params.voucherType) {
+
```

Comment:
if (params.voucherType) {
	if (params.voucherType === 'sales' || params.voucherType === 'debit note' || params.voucherType === 'credit note') {
		this.selectedVoucherType = params.voucherType;
	} else {
		this.selectedVoucherType = params.selectedType || params.voucherType;
	}
}